Podfile

```diff
@@ -31,13 +31,32 @@ target 'TestApp' do
   pod 'AEPLifecycle'
   pod 'AEPSignal'
   pod 'AEPAssurance'
-  pod 'AEPEdge'
-  pod 'AEPEdgeConsent'
+  pod 'AEPEdge', :git => 'https://github.com/adobe/aepsdk-edge-ios.git', :branch => 'staging'
```

Comment:
Should we change back to AEPEdge since going to Main?

Reply:
We don't have it main right now. We will update it once we release all extension with tvOS support.